input/pagecontent/terminology.md

```diff
@@ -66,6 +67,7 @@ The list below shows the code systems used in the value sets above.
 |[NameUse](https://hl7.org/fhir/R4/codesystem-name-use.html)|[NameUse](https://hl7.org/fhir/R4/valueset-name-use.html)|FHIR|
 |[RequestIntent](https://hl7.org/fhir/R4/codesystem-request-intent.html)|[RequestIntent](https://hl7.org/fhir/R4/valueset-request-intent.html)|FHIR|
 |[RequestStatus](https://hl7.org/fhir/R4/codesystem-request-status.html)|[AU eRequesting RequestStatus](ValueSet-au-erequesting-request-status.html)|FHIR|
+|[SexParameterForClinicalUseCS](http://terminology.hl7.org/CodeSystem/sex-parameter-for-clinical-use)|[SexParameterForClinicalUse](http://terminology.hl7.org/ValueSet/sex-parameter-for-clinical-use)|THO|
```

Comment:
1. Please change 'SexParameterForClinicalUseCS' to match the code system title 'Sex Parameter For Clinical Use'
2. Please change 'SexParameterForClinicalUse' to match the value set title 'Sex Parameter for Clinical Use'.
3. Please change 'THO' to 'HL7 Terminology'.

Reply:
done